contrib/guix/guix-clean

```diff
@@ -15,7 +15,7 @@ source "$(dirname "${BASH_SOURCE[0]}")/libexec/prelude.bash"
 ###################
 
 ################
-# Required non-builtin commands should be invokable
+# Required non-builtin commands should be invocable
```

Comment:
is that really a typo? invokable seems to exist: https://en.wiktionary.org/wiki/invokable (and that site lists invocable as an alternative spelling)